app/ui/legacy/src/main/java/com/fsck/k9/activity/AccountList.java

```diff
@@ -70,25 +70,13 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
     public void populateListView(List<Account> realAccounts) {
         List<BaseAccount> accounts = new ArrayList<>();
 
-        if (displaySpecialAccounts() && !K9.isHideSpecialAccounts()) {
```

Comment:
Hm, I didn't consider screens based on `AccountList` when creating #5020. For now we should probably keep the current behavior.

Reply:
For clarification, you want to keep the unified Inbox account in accounts list object, you just want me to find another way of keeping it out of the UI.  I think I can do that.

Reply:
No, just don't touch these screens. Really, #5020 is about the drawer only.

Reply:
I think i'm not writing precisely.  You want to maintain accountlist behavior as it was prior to this patch, but still want to remove the unified inbox account from the "drawer" account selector spinner.  you don't want to impact any other "screens" that may be using accountlist at this time.  did I get it right?

Reply:
I think "account list" vs. `AccountList` was tripping me up. So yes, change the drawer, don't touch anything related to the `AccountList` class, i.e. the `ChooseAccount` and `LauncherShortcuts` activities.

Reply:
now I see what I did.  I just assumed that AccountList.accounts were somehow making it's way over to the drawer, but the injection is of some other list of accounts.   it seems that specialaccounts is a somewhat incomplete metaphor.  perhaps I'm mistaken.